mk/submodule.mk

```diff
@@ -49,7 +51,21 @@ else
          cmake -DCMAKE_INSTALL_PREFIX=. .. && \
          cmake --build . --target install
 endif
-
-$(LIBRABBITMQ_OBJS):
 	cd $(LIBRABBITMQ_PATH)/build && \
          cmake --build .
+
+# Submodules
+SUBS := $(LIBTUV_PATH)/.git
```

Comment:
Ditto. We might retrieve packages from archives instead of Git repositories.

Reply:
Got it.

Reply:
Done.